python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -420,15 +424,15 @@ def avg_pooling(attrs, inputs, cls):
 
 
 def max_pooling(attrs, inputs, cls):
-    """ Average pooling"""
+    """ Max pooling"""
     new_attrs = translation_utils._fix_attribute_names(attrs,
                                                        {'kernel_shape': 'kernel',
                                                         'strides': 'stride',
                                                         'pads': 'pad',
                                                        })
 
     new_attrs = translation_utils._add_extra_attributes(new_attrs,
-                                                        {'pool_type': 'avg',
+                                                        {'pool_type': 'max',
```

Comment:
Thanks for catching,  but looking at the code looks like  'pool_type: 'max' is actually redundant code.  I think we can get rid of that

Reply:
True true the type is passed to _fix_pooling function. I will update to remove the pool_type field here.